src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -432,6 +446,27 @@ public List<String> run() {
         }
     }
 
+    private void verifyTLSVersion(final String settings, final List<String> configuredProtocols) {
+        for (final var tls : configuredProtocols) {
+            if (tls.equalsIgnoreCase("TLSv1")) {
+                deprecationLogger.deprecate(
+                    settings,
+                    "The '{}' setting contains TLSv1 protocol version which was deprecated since 2021 (RFC 8996). "
+                        + "Support for it will be removed in the next major release.",
+                    settings
+                );
+            }
+            if (tls.equalsIgnoreCase("TLSv1.1")) {
```

Comment:
`if/else` would work better here, if `tls == TLSv1`, no need to check for `TLSv1.1` after
```
if (tls.equalsIgnoreCase("TLSv1")) {
  ...
} else if (tls.equalsIgnoreCase("TLSv1.1")) {
  ...
}
```

Reply:
addressed